tests/python/contrib/test_hexagon/test_conv2d_blocked.py

```diff
@@ -0,0 +1,474 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import sys
+
+import tvm
+from tvm import te
+from tvm import topi
+from tvm.topi import testing
+from .infrastructure import ceildiv
+from .infrastructure import get_packed_layout
+from .infrastructure import build_and_run
+
+import numpy as np
+import pytest
+
+
+def conv2d_logical(
+    shape_nhwc,
+    shape_kcrs,
```

Comment:
comment: we haven't been using KCRS convention more commonly used by Nvidia etc. but instead have been using the OIHW notation for better or worse. Doing a quick search for the use of KCRS pointed me to your texture codegen work :)

Reply:
Yeah, I prefer KCRS when doing conv2d because it makes it clear that the C in the filter is the same C in the input activation, and the K is the same K in the output activation. It also disambiguates the HW in the input (NHWC) and the filter (OIHW) by instead using R and S. When the filter shape is discussed in isolation, it's okay to use OIHW, but when writing out compute definitions for convolution, nhwc and oihw together become really messy in my opinion.

Reply:
I can convert the conv2d definitions over to OIHW for consistency. I actually did in anticipation of someone making this comment, but I reverted it because I became confused when reading the schedule. :) Too many axes like `Oi`, `Ii`, `Iii`, `H_input`, `H_filter`, `W_inp`, `W_filt` floating around. That said, I don't have strong feelings on it either way.

Reply:
Good point on the notation being less ambiguous. I'd recommend using OIHW notation for API (function names and argument names) but for axes, I think the KCRS is nice because it avoids the ambiguity on W and H etc.

Reply:
Good suggestion, I updated the API as you suggested!